board/app/controllers/user_controller.php

```diff
@@ -1,92 +1,129 @@
 <?php
 
-class UserController extends AppController{
+class UserController extends AppController 
+{
+
+const LOGIN_PAGE = 'login';
+const HOME_PAGE = 'home';
+const REGISTER = 'register';
+const REGISTER_END = 'register_end';
+const EDIT = 'edit';
+const EDIT_END = 'edit_end';
```

Comment:
Done and applied to all po. Thank you :bow:
--> https://github.com/maragatchalian/FoodFacts/commit/914551051462cd2000b0bb73be4f8df86e178ace